server/bingo/tests/test_inactive_user_deletion.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+from bingo.models import User
+from freezegun import freeze_time
+from django.utils import timezone
+from datetime import timedelta
+from ..tasks import delete_inactive
+
+
+class InactiveUserDeletionTestCase(TestCase):
+    def setUp(self):
+        with freeze_time(timezone.now() - timedelta(days=2)):
+            self.old_inactive_user = User.objects.create(
+                username="user1", password="verysecure123", email="email1@example.com", is_active=False)
+        with freeze_time(timezone.now() - timedelta(hours=1)):
+            self.recent_inactive_user = User.objects.create(
+                username="user2", password="verysecure123", email="email2@example.com", is_active=False)
+        self.new_inactive_user = User.objects.create(
+            username="user3", password="verysecure123", email="email3@example.com", is_active=False)
+        with freeze_time(timezone.now() - timedelta(days=2)):
+            self.old_active_user = User.objects.create(
+                username="user4", password="verysecure123", email="email4@example.com")
+        with freeze_time(timezone.now() - timedelta(hours=1)):
+            self.recent_active_user = User.objects.create(
+                username="user5", password="verysecure123", email="email5@example.com", is_active=False)
+        self.new_active_user = User.objects.create(
+            username="user6", password="verysecure123", email="email6@example.com", is_active=False)
```

Comment:
Not sure if this is an oversight or intended but `recent_active_user` and `new_active_user` are created as inactive (`is_active=False`). 

Active users should be `is_active=True` right?